src/app/shared/search/search-filters/search-filter/search-filter.component.html

```diff
@@ -1,14 +1,15 @@
 <div class="facet-filter d-block mb-3 p-3" *ngIf="active$ | async"
-     [id]="regionId" [attr.aria-labelledby]="toggleId" [ngClass]="{ 'focus': focusBox }" role="region">
+     [id]="regionId"  [ngClass]="{ 'focus': focusBox }" >
   <button (click)="toggle()" (focusin)="focusBox = true" (focusout)="focusBox = false"
           class="filter-name d-flex" [attr.aria-controls]="regionId" [id]="toggleId"
           [attr.aria-expanded]="!(collapsed$ | async)"
           [attr.aria-label]="(((collapsed$ | async) ? 'search.filters.filter.expand' : 'search.filters.filter.collapse') | translate) + ' ' + (('search.filters.filter.' + filter.name + '.head') | translate | lowercase)"
           [attr.data-test]="'filter-toggle' | dsBrowserOnly"
   >
-    <span class="h4 d-inline-block text-left mt-auto mb-auto">
+    <label class="h5 d-inline-block mb-0">
```

Comment:
It's unclear to me why this is now a `<label>` as this now throws accessibility warnings when using the WAVE plugin,  It tells me that this label is "orphaned" because it is not associated with any form element.  Could we just turn this back into a `<span>`?  It's unclear why we've changed this.

Reply:
As mentioned last week, it's unclear why this has been turned into a `<label>` as that causes this page to throw accessibility warnings when using the WAVE plugin. It tells me that this label is "orphaned" because it is not associated with any form element. Could we just turn this back into a `<span>`? It's unclear why we've changed this.